backend/framework/bin/qlf_pipeline.py

```diff
@@ -244,6 +248,19 @@ def ingest_parallel_qas(self):
         logger.info("Total runtime: %s." % (self.data.get('duration') + duration_ingestion))
         logger.info("ExpID {} is ready for analysis".format(self.data.get('expid')))
 
+    def generate_qa_tests(self, process_id):
+        qa_tests = list()
+        for job in self.models.get_jobs_by_process_id(process_id):
```

Comment:
I think the request in `self.models.get_jobs_by_process_id(process_id)` is not necessary. You can get the information in `self.data.get('cameras')`. Look at line 211.